token-locker/src/lib.rs

```diff
@@ -170,10 +170,10 @@ impl Contract {
         let event = EthUnlockedEvent::from_log_entry_data(&proof.log_entry_data);
         assert_eq!(
             event.bridge_address,
-            self.bridge_address,
+            self.factory_address,
             "Event's address {} does not match locker address of this token {}",
             hex::encode(&event.bridge_address),
-            hex::encode(&self.bridge_address),
+            hex::encode(&self.factory_address),
         );
```

Comment:
Let's rename the variable in the event too

Reply:
Fixed c112a8c38f43c876660c52e06a601a6ee01281f6